package.json

```diff
@@ -8,9 +8,15 @@
     "*.d.ts"
   ],
   "scripts": {
-    "bundle": "gulp bundle",
-    "doc": "gulp doc",
-    "test": "gulp test"
+    "bundle": "npm run clean && npm run compile && mv _build/src/** . && node _build/tools/bundle.js",
+    "clean": "rm -rf _build coverage doc",
+    "compile": "tsc",
+    "coverage": "cat ./coverage/lcov.info | coveralls",
+    "lint": "find src test -name \"*.ts\" | sed 's/^/--file=/g' | xargs tslint",
+    "pretest": "npm run compile -- --sourceMap & npm run lint",
+    "test": "istanbul cover _mocha -- --reporter $npm_package_config_reporter --slow 10 --ui bdd --recursive _build/test",
```

Comment:
Does the `$npm_package_config_reporter` variable come in the environment for free? Also, want to put quotes or braces around it? I can see it doesn't have spaces in it, but if it did (or if someone copied/pasted this command)...